### PR TITLE
Feature/001 workspace add

### PR DESCRIPTION
**Pull request by shabnamadil**

### CODE REVIEW

**Comment by shabnamadil** [comment]
Everything is ok

File: project/apps/workspace/models/workspace_members.py
Comment (ilgarhasanzada) [inline project/apps/workspace/models/workspace_members.py:1]: bu kod dogru caslisir

**Review by ilgarhasanzada — COMMENTED** [review]
All reviews is checked

**Comment by ilgarhasanzada** [comment]
bu commit review edildi

**Review by ilgarhasanzada — COMMENTED** [review]
rewview edildi

**Pull request merged**